docs/source/release/v6.4.0/Indirect/Algorithms/Bugfixes/33064.rst

```diff
@@ -0,0 +1,4 @@
+The Abins algorithm has reworked to account for q-dependence
+correctly when forming 3+ phonon events by self-convolution. This
+changes the results obtained when using the "Autoconvolution"
+option, which now results in the expected high-frequency "wing".
```

Comment:
would it be worth including a plot of before and after the change to highlight the new wing?

Reply:
Good idea! Is there any guidance on how that should be done in the new system? Would a file in this directory and a relative link in the RST work?

Reply:
@sf1919 is this in your docs?

Reply:
Ok, done! Not sure how to preview it, I guess problems if any will come up at release time?

Reply:
Leave it as is. If there are any problems it will be picked up once we have the compilation process in place. If changes do need to be made I will make them and update documentation accordingly.